07-modeling.Rmd

```diff
@@ -203,18 +207,18 @@ recs_in %>%
   theme_bw() 
 ```
 
-The model is fit below with electricity expenditure as the outcome. Some textbooks use the term simple linear regression when a linear model has one continuous independent variable.
+The model is fit below with electricity expenditure as the outcome. Some textbooks use the term simple linear regression when a linear model has one continuous independent variable. 
 
 ```{r}
 #| label: model-slr-examp
 m_electric_sqft <- recs_des %>%
-   svyglm(design=.,
-          formula=DOLLAREL~TOTSQFT_EN,
-          na.action=na.omit)
+  svyglm(design = .,
+         formula = DOLLAREL ~ TOTSQFT_EN,
+         na.action = na.omit)
 tidy(m_electric_sqft)
 ```
 
-In the output above, we can see the estimated coefficients (`estimate`), estimated standard errors of the coefficients (`std.error`), the t-statistic (`statistic`), and the p-value for each coefficient. In these results, we can say that, on average, for every additional square foot of house size, the electricity bill increases by `r (tidy(m_electric_sqft) %>% filter(term=="TOTSQFT_EN") %>% pull(estimate) %>% signif(3))*100` cents and that square footage is significantly associated with electricity expenditure. This is a very simple model, and there are likely many more factors in electricity expenditure, including the type of cooling, number of appliances, location, and more. 
+In the output above, we can see the estimated coefficients (`estimate`), estimated standard errors of the coefficients (`std.error`), the t-statistic (`statistic`), and the p-value for each coefficient. In these results, we can say that, on average, for every additional square foot of house size, the electricity bill increases by `r (tidy(m_electric_sqft) %>% filter(term=="TOTSQFT_EN") %>% pull(estimate) %>% signif(3))*100` cents and that square footage is significantly associated with electricity expenditure. This is a very simple model, and there are likely many more factors in electricity expenditure, including the type of cooling, number of appliances, location, and more.
```

Comment:
True, but we can add the usefulness of this simple model, and maybe either how to use it or recommended next steps.